src/main/kotlin/no/nav/syfo/client/pdfgen/PdfGenClient.kt

```diff
@@ -55,7 +58,7 @@ class PdfGenClient(
         documentComponentDTOList: List<DocumentComponentDTO>,
     ): ByteArray? = getPdf(
         callId = callId,
-        payload = documentComponentDTOList,
+        payload = documentComponentDTOList.sanitizeForPdfGen(),
```

Comment:
Påminnelse og Retur er jo meldingstyper der det er en fast tekst skrevet av oss. Egentlig trenger man ikke å gjøre dette for de meldingene, men jeg og Anders tenkte også at det like greit kunne være på alle. Vi tror ikke kostnaden er så stor at det gjør noe 🤔